packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -168,7 +168,10 @@ describe('logs', () => {
       sendLog(DEFAULT_MESSAGE, {})
 
       expect(server.requests.length).toEqual(0)
-      expect(sendSpy).toHaveBeenCalled()
+      expect(sendSpy).toHaveBeenCalledOnceWith(
+        '{"eventType":"log",' +
+          '"event":{"service":"service","session_id":"session-id","status":"info","message":"message"}}'
+      )
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏to make those tests less brittle in case the structure or the content of the message changes, it could be interesting to only check what we are interested in, I'd say the `eventType` and something specific to the log/event sent by the test. Something like:

```js
expect(msg).toContain('"eventType":"log"')
expect(msg).toContain('"message":"message"')
```

Reply:
I would go one step further by parsing the message and using jasmine assertions on the resulting object

Reply:
Is there any advantages of parsing the message? 
To me, it seems okay to test a string if the expected message output is a string.

Reply:
I've done the change